src/Theme.tsx

```diff
@@ -24,10 +24,10 @@ const darkTheme: ThemeOptions = {
   palette: {
     mode: 'dark',
     primary: {
-      main: '#20ba4e'
+      main: '#fd8ccc'
```

Comment:
Revert theme changes.

A more extensive UI customization system would be pretty cool too for a future PR :wink:

Reply:
It would, pushing revert changes currently